frontend/src/pages/profile/ProfileView.tsx

```diff
@@ -0,0 +1,292 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  Row,
+  Col,
+  Card
+} from "react-bootstrap";
+import { observer } from "mobx-react-lite";
+import './ProfileView.scss';
+import 'react-toastify/dist/ReactToastify.css';
+import { showToast, copyProfileUrlToClipboard } from "../../utils";
+import UserProfile from "../../model/UserProfile"
+import Session from "../../model/Session"
+import SearchResult from "../search/SearchResult";
+
+type InputFieldProps = {
+  userProfile: UserProfile;
+  label: string;
+  tooltip: string;
+  placeholder: string;
+  attributeName: string;
+  icon: string;
+}
+
+const InputField = observer((props: InputFieldProps) => {
+  let valueField: string = props.userProfile.getAttribute(props.attributeName, "");
+  return (
+    <form className="row g-3 mb-15">
+      <div className="col-md-12">
+        <label>
+          <i className={"mdi me-1 vertical-middle " + props.icon}></i>
+          {props.label}:
+{/*           <OverlayTrigger */}
+{/*             placement="right" */}
+{/*             delay={{ show: 50, hide: 400 }} */}
+{/*             overlay={(props) => ( */}
+{/*               <Tooltip id="button-tooltip" {...props}> */}
```

Comment:
Same doubt as before about tooltips. In the meantime, I choose the current method of using cites below the field.